assets/app/scripts/filters/capitalize.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+angular.module('openshiftConsole')
+  .filter('capitalize', function() {
+    return function(input) {
+      return _.capitalize(input);
+    };
+  });
```

Comment:
I made a note already to PR & break utils.js down.... at least into strings.js, arrays.js, etc.  If not that, a folder for `/strings`, `/arrays` and individual files, etc.  A big file of all the things makes me sad.  :cry: